client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -395,8 +395,8 @@ export function YourBrowserSupportToolkit() {
         <h2>Modern CSS in the real world : Part two</h2>
       </header>
       <p className="article-lead">
-        Viverra neque eget tellus, gravida pellentesque non. Purus, venenatis
-        sit ac nam nec. At vitae facilisi in mi amet aliquam.
+        In this article discover the resources available, to help you develop a 
+        site that will perform well across browsers and devices.
```

Comment:
Could you remove the period at the end of the tagline?

Reply:
Do you mean so it ends with `... well across browsers and devices` instead of `... well across browsers and devices.`?
So a sentence without a full stop?
But the other article lead in the other article as a trailing period on its tagline.

Reply:
The last period shouldn't be there. I haven't seen the other article page yet, but the designs should match the mock-ups linked on the Trello card: https://trello.com/c/TPJwawMo/87-getting-more-market-feedback-on-content

Reply:
So no period at the end of either taglines